### PR TITLE
Populate databases with examples

### PR DESCRIPTION
**Pull request by paulmis**
I've populated Firestore and Cloud Storage with output examples. I've made some changes to the [**schema** ](https://lucid.app/lucidchart/099c67af-b54e-4aaf-9c7d-d11d0909d060/edit?page=0_0&invitationId=inv_3b33311a-b7cd-49ec-aba3-9a85df7d53c8#) and [**bucket layout**](https://app.clickup.com/9004004983/v/dc/8cawdkq-1447/8cawdkq-1367):
- renamed runs to scans
- added pipeline tags (scan, auxiliary)
  - scan pipelines produce final output to display in the app
  - auxiliary pipelines aid in the data collection process, and their output should largely be ignored by the frontend
- moved indices and insights from the pipeline to scans:
  - pipelines dump their _pipeline data object (PDO, **[see here](https://github.com/GDSC-Delft-Dev/apa/tree/main/src/backend)**)_ to the pipeline collection 
  - when a scan is finished, the pipeline dumps the frontend-relevant data into the scan collection
- grouped scans by fields rather than users

**Please see if you agree with all these changes!!**

The example data is here:
- **[Firestore](https://console.cloud.google.com/firestore/databases/-default-/data/panel/scans/c6bd6bf3-8ea9-40f3-ac73-cd46521a5270?project=terrafarm-378218)** (scan c6bd6bf3-8ea9-40f3-ac73-cd46521a5270)
- **[Cloud Storage](https://console.cloud.google.com/storage/browser/terrafarm-scans/NlDfTwwMaYaVB1eTYYIkAdSDkMg1/03-2022/c6bd6bf3-8ea9-40f3-ac73-cd46521a5270;tab=objects?project=terrafarm-378218&prefix=&forceOnObjectsSortingFiltering=false)** 

One important thing is that since we're supporting cameras that potentially don't capture all 3 RGB channels, the RGB index outputs the "default" image. However, that module might not be in a given pipeline, so the frontend needs to account for that.

Aaand regarding image sizes, they can grow very large - the example full-size RGB image is 0.5GB. We will downsize and compress the images on the backend to make it easier for you to display thumbnails and avoid progressive loading for the moment. What sizes (resolution, weight) do you guys want us to produce @nadinekuo @RD211?

- [x] Set real boundary coordinates for the example data field
- [x] Add a `Mat::fread` overload for purely grayscale loading

### CODE REVIEW

**Comment by nadinekuo** [comment]
Thanks Paul! LGTM. As for image size and weight, I'd say around 1024 x 1024 px (around  5 MB) to have decent quality but keep the app a bit responsive ;)

**Review by nadinekuo — APPROVED** [review]
Thanks Paul! LGTM. As for image size and weight, I'd say around 1024 x 1024 px (around 5 MB) to have decent quality but keep the app a bit responsive ;)

**Comment by nadinekuo** [comment]
By "the RGB index outputs the "default" image", do you mean the plain satellite image? Is this different from keeping the current satellite map that Google Maps provides currently?
Also, is the NDVI .tif image supposed to be in grayscale?

**Comment by paulmis** [comment]
@nadinekuo 

> As for image size and weight, I'd say around 1024 x 1024 px (around 5 MB) to have decent quality but keep the app a bit responsive ;)

The downscaled image is 5000x4000 and is 28.5MB, which comes out at around 1.5MB/MP. Do you think this is good, or should it be a bit bigger?

> By "the RGB index outputs the "default" image", do you mean the plain image? 

Yeah, as in the normal color image.

> In case the pipeline does not contain the "RGB" module, would we just rely on the satellite images provided by Google Maps?

Yes

> Also, is the NDVI .tif image supposed to be in grayscale?

Yeah, as far as I can tell that's the format to store grayscale. What you can do is color it using an arbitrary color scheme (e.g. for NDVI is `RdYlGn` to transform it into a color scale. Can you make an issue for that for the frontend?

**Comment by nadinekuo** [comment]
Also, the bucket layout is as follows: `terraform-scans/{field}/{MM-YYYY}/{scan}/pipelines/{pipeline}` whereas the `{field}` is `{user}` at the moment?

**Comment by nadinekuo** [comment]
> The downscaled image is 5000x4000 and is 28.5MB, which comes out at around 1.5MB/MP. Do you think this is good, or should it be a bit bigger?

I'm afraid that even the downscaled image is too large, but we can resize on the frontend after fetching them

**Comment by paulmis** [comment]
> Also, the bucket layout is as follows: terraform-scans/{field}/{MM-YYYY}/{scan}/pipelines/{pipeline} whereas {field} is {user} at the moment?

Might have forgotten to change it, please feel free to switch it to the field id.

> I'm afraid that even the downscaled image is too large (users typically don't even have close to that many pixels), but we can resize on the frontend after fetching them

No, I meant I will downsize it to the target size/resolution at the backend, but do you want me to do it according to the 5MB size or 1MP resolution?

**Comment by nadinekuo** [comment]
> No, I meant I will downsize it to the target size/resolution at the backend, but do you want me to do it according to the 5MB size or 1MP resolution?

I'd say around 5MB, as 28.5 MB is most likely gonna lead to a drastic slowdown 😅 
The ideal setting would be having different resolutions per zoom level but that is hard

File: src/backend/main.py [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+from pipeline.mat import Mat, Channels
+import os
+import numpy as np
 
 def main():
     """Main entry point."""
 
     # Get test data
-    imgs = [Mat.read(file) for file in glob.glob("pipeline/test/data/mosaicing/farm/D*.JPG")]
+    paths = glob.glob(f"C:/Users/jesuschrist/Downloads/dataverse_files/dataset/2017-06-21_Wageningen_TestField_Sequoia_F3_7_55/Level_2/*.tif")
```
Comment (mirceatlx) [inline src/backend/main.py:11]: Try to remove hard-coded path:)

**Review by mirceatlx — COMMENTED** [review]
Looks good, I will approve after we resolve all conversations.

File: src/backend/main.py (hunk 1 above)
Comment (paulmis) [reply src/backend/main.py]: Done

**Review by mirceatlx — APPROVED** [review]

**Pull request merged**